lom/src/plugins/plugins_files/GenericPluginDetection.go

```diff
@@ -29,7 +29,9 @@ func NewGenericPluginDetection(...interface{}) plugins_common.Plugin {
 
 func init() {
```

Comment:
Can we rename it as GenericPluginDetection_test.go. 

BTW, what is the purpose of plugin_files ?

Please sync with Prithvi. He creates a dir per plugin as linkCRC. You may create a subdir test-plugins and move this there.

Reply:
yes. Initial idea when I started is to have all generic plugins here. I will move it or rename this folder.

Reply:
TODO